src/DDTimingUtils.cpp

```diff
@@ -14,8 +14,17 @@ void IgnoreClockAfterResetTimer::process(float deltaTime) {
 	}
 };
 
+void SequelClockTracker::initializeRows() {
+	for(int i = 0; i < numRows; i++) {
+		currentStepTracker.push_back(0);
+		gatesSinceLastStepTracker.push_back(0);
+		divideTracker.push_back(1);
+		hasPulsedThisStepTracker.push_back(false);
+	}
+};
+
 void SequelClockTracker::nextClock() {
-	for(int i = 0; i < numSteps; i++) {
+	for(int i = 0; i < numRows; i++) {
```

Comment:
This line was necessary to fix the exit crash fault, I was referencing index on the vectors that didn't exist because I was incorrectly iterating through numSteps instead of numRows